compliance/main.rego

```diff
@@ -15,15 +15,19 @@ findings = f {
 	# iterate over activated benchmarks
 	benchmarks := [key | data.activated_rules[key]]
 
+	rego := compliance
```

Comment:
1. is it necessary to duplicate in both rules or can this assignment be outside of them?
2. rename `rego` to something more clear